index.html

```diff
@@ -237,7 +237,7 @@ <h2>Get in Touch</h2>
             <div class="row">
                 <div class="col-sm-5 copyright">
                     <p>
-                        Copyright &copy; 2017 YOUR NAME
+                        Copyright &copy; 2019 Sagar Kunwar Chhetri
```

Comment:
I think these changes will take away from portfolio customization. Ideally, you should fork the project and include your information there.